posthog/temporal/data_imports/pipelines/postgres/__init__.py

```diff
@@ -37,6 +37,31 @@ def postgres_source(
     return db_source
```

Comment:
Rename this folder to databases? or something better than postgres. won't be obvious snowflake is in here too